### PR TITLE
removed create facility option for existing user while searching

### PR DESCRIPTION
**Pull request by AJIL-PAUL**
Issue :https://github.com/coronasafe/care_fe/issues/837

Removed create facility option for existing user while searching facilities which are not existing.
Also changed search tab such that the icon is in right side with a working clear button

### CODE REVIEW

File: src/Common/constants.tsx [hunk 1]
```diff
@@ -224,6 +223,7 @@ export const ADMITTED_TO = [
   "ICU with Non Invasive Ventilator",
   "ICU with Oxygen Support",
   "ICU with Invasive Ventilator",
+  "BED WITH OXYGEN SUPPORT",
```
Comment (bodhish) [inline src/Common/constants.tsx:226]: Do we have this bed type in backend?

**Review by bodhish — APPROVED** [review]
The changes look good, I am just confused with the key. Could you check the backend and confirm the key?

File: src/Common/constants.tsx (hunk 1 above)
Comment (ajil-paul) [reply src/Common/constants.tsx:226]: I added based on issue  #871 
But need to check the backend

Comment (bodhish) [reply src/Common/constants.tsx:226]: @vigneshhari can you confirm this?

**Review by vigneshhari — APPROVED** [review]

File: src/Common/constants.tsx (hunk 1 above)
Comment (vigneshhari) [reply src/Common/constants.tsx:226]: will confirm soon

**Pull request merged**